README.md

```diff
@@ -25,14 +23,16 @@ Two additional ways, besides Gitter or Slack chat above, to contact us:
 ### Getting Latest Executable: i18n4go
 --------------------------------------
 
-Assuming you have a valid [Golang 1.4.2](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
+Assuming you have a valid [Golang 1.19.5](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
 
 ```
 $ go get github.com/maximilien/i18n4go/i18n4go
```

Comment:
Nit: Using `go get` is deprecated for installing binaries.  It is recommend by google to use `go install` to install binaries and compiled packages ([doc](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies))

Also, as of 1.18 `go get` will not build packages so it is better to use `go install` in the long run. ([blog](https://go.dev/doc/go-get-install-deprecation))

```suggestion
$ go install github.com/maximilien/i18n4go/i18n4go
```